app/controllers/course-overview.ts

```diff
@@ -24,6 +24,14 @@ export default class CourseOverviewController extends Controller {
     }
   }
 
+  get completedStages() {
+    if (!this.model.course) {
```

Comment:
Under what circumstances would `model.course` be non-null? Wouldn't it always be a `CourseModel`?